astropy/modeling/tests/test_constraints.py

```diff
@@ -23,6 +23,9 @@
 except ImportError:
     HAS_SCIPY = False
 
+# Module level random state for reproducible results.
+rsn = RandomState(1234567890)
```

Comment:
Module level won't work if people choose to run a single test, I think we need to set the random state inside each test that generates random numbers